Tests/LegacyUnit/FrontEnd/EventEditorTest.php

```diff
@@ -2477,4 +2519,124 @@ public function getPreselectedOrganizerForTwoAvailableOrganizersReturnsZero()
             $this->subject->getPreselectedOrganizer()
         );
     }
+
+    /*
+     * Utility functions
+     */
+
+    /**
+     * Adds an instance to the Typo3 instance FIFO buffer used by `GeneralUtility::makeInstance()`
+     * and registers it for purging in `tearDown()`.
+     *
+     * In case of a failing test or an exception in the test before the instance is taken
+     * from the FIFO buffer, the instance would stay in the buffer and make following tests
+     * fail. This function adds it to the list of instances to purge in `tearDown()` in addition
+     * to `GeneralUtility::addInstance()`.
+     *
+     * @param string $className
+     * @param mixed $instance
+     *
+     * @return void
+     */
+    private function addMockedInstance(string $className, $instance)
```

Comment:
We have all this code twice now (also for the other PR). Please move the utility functions to a trait and add a testcase for the trait utility functions.

Reply:
The trait then should have an `@mixin TestCase` annotation.